db/migrate/20181023082013_add_user_id_to_workspace.rb

```diff
@@ -0,0 +1,6 @@
+class AddUserIdToWorkspace < ActiveRecord::Migration[5.2]
+  def change
+    add_column :workspaces, :user_id, :integer
```

Comment:
prefer [built-in](https://apidock.com/rails/ActiveRecord/ConnectionAdapters/SchemaStatements/add_reference) methods 
`add_reference(:workspaces, :user)`

Reply:
as for me this is optional. Personally I prefer old-style syntax which is more readable